src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -121,11 +137,12 @@
      * @param { DOM } element
      */
     var isBackground = function isBackground(element) {
-      return $(element).hasClass('BookReader')
+      var isBackground = $(element).hasClass('BookReader')
```

Comment:
Having a local variable with the same name as the variable that contains this scoped function is really odd. Can this or the function be renamed?

Reply:
thanks for catching, my update got swallowed up by a rebase